### PR TITLE
Reportspermissions

### PR DESCRIPTION
**Pull request by grioux-neo**
Adjusted reports permissions and placed comments for future expansion to flight-scoped reports.

### CODE REVIEW

File: packages/client/src/pages/admin/pluggables/Reports.tsx [hunk 1]
```diff
@@ -64,9 +66,28 @@ interface ReportsWidgetViewState {
...
+
+// const hasAllowedDutyPosition = hasOneDutyPosition([
+// 	'Cadet Flight Commander',
+// 	'Cadet Flight Sergeant',
+// 	'Cadet Commander',
+// 	'Cadet Deputy Commander',
+// 	'Cadet Executive Officer',
+// 	'Deputy Commander For Cadets',
+// ]);
+
```
Comment (r-a303931) [inline packages/client/src/pages/admin/pluggables/Reports.tsx:86]: Can be deleted, it's located in FlightContact.tsx

```suggestion

```

File: packages/client/src/pages/admin/pluggables/Reports.tsx [hunk 2]
```diff
@@ -64,9 +66,28 @@ interface ReportsWidgetViewState {
...
 export const shouldRenderReports = (props: PageProps): boolean =>
 	!!props.member &&
-	hasPermission('PromotionManagement')(Permissions.PromotionManagement.FULL)(props.member);
+	hasPermission('PromotionManagement')(Permissions.PromotionManagement.FULL)(props.member); // ||
+	// hasAllowedDutyPosition(props.member)
```
Comment (r-a303931) [inline packages/client/src/pages/admin/pluggables/Reports.tsx:90]: Same as before

```suggestion
	hasPermission('PromotionManagement')(Permissions.PromotionManagement.FULL)(props.member);
```

File: packages/client/src/pages/admin/pluggables/Reports.tsx [hunk 3]
```diff
@@ -132,21 +153,19 @@ export const ReportsWidget = withFetchApi(
...
-										>
-											SQR 60-1 Cadet status report
-										</Button>
-										<br />
-									</>
-								) : null}
+									(
+										hasPermission('PromotionManagement')(Permissions.PromotionManagement.FULL)(this.props.member) ||
+										hasAllowedDutyPosition(this.props.member)
+									) ? ( */}
```
Comment (r-a303931) [inline packages/client/src/pages/admin/pluggables/Reports.tsx:161]: Same as before

**Review by r-a303931 — CHANGES_REQUESTED** [review]

File: packages/client/src/pages/admin/pluggables/Reports.tsx (hunk 1 above)
Comment (grioux-neo) [reply packages/client/src/pages/admin/pluggables/Reports.tsx:86]: It's a placeholder.  I'm not done with it yet.  I'll come back to it.

File: packages/client/src/pages/admin/pluggables/Reports.tsx (hunk 2 above)
Comment (grioux-neo) [reply packages/client/src/pages/admin/pluggables/Reports.tsx:90]: It's a placeholder.  I'm not done with it yet.  I'll come back to it.

File: packages/client/src/pages/admin/pluggables/Reports.tsx (hunk 3 above)
Comment (grioux-neo) [reply packages/client/src/pages/admin/pluggables/Reports.tsx:161]: It's a placeholder.  I'm not done with it yet.  I'll come back to it.

**Pull request closed without merging**